nucypher/utilities/sandbox/middleware.py

```diff
@@ -107,6 +110,17 @@ def put_treasure_map_on_node(self, node, map_id, map_payload):
                                     data=map_payload, verify=certificate_filepath)
         return response
 
+    def revoke_arrangement(self, ursula, revocation):
+        mock_client = self._get_mock_client_by_ursula(ursula)
+        response = mock_client.delete('http://localhost/kFrag/{}'.format(
+                                      revocation.arrangement_id.hex()),
+                                      data=bytes(revocation))
+        
+        if response.status_code != 200:
+            if response.status_code != 404:
```

Comment:
What about replacing these two lines by `if response.status_code not in (200, 404):`

Another options is `if response.status_code != 200 and response.status_code != 404:`, although it's more verbose.